src/Tests/DependencyInjection/SurfnetYubikeyApiClientExtensionTest.php

```diff
@@ -15,7 +15,7 @@ class SurfnetYubikeyApiClientExtensionTest extends TestCase
      * @test
      * @group DependencyInjection
      */
-    public function verification_service_can_be_loaded()
+    public function verificationServiceCanBeLoaded()
```

Comment:
I do not think this makes readability any better. Most of our projects allow for snake casing test method names to allow for better readable test names. Can you revert this and fix it in phpmd? Or alternatively, prohibit phpmd from sniffing the tests namespace. (which i would discourage)

Reply:
@MKodde 
This is a bummer. Small change but... I deliberately changed this becase of this project constraints. If the project constraints change (this is not stepup as far as I know) I like to know this upfront (preferablye with a organisation wide standard config of all QA tools).

Reply:
I leave it for now, we have bigger taksks to do, otherwise I have to rename every test by hand. Future work.

Reply:
I believe all projects have a PHPMD setting to allow snake case function names for tests.

> Otherwise I have to rename every test by hand
Reverting a commit is not that much work right?

I'm OK with leaving this for now. But I do feel this is a regression.